### ISSUE TITLE
Thumbnail should be hidden for screen readers

### ISSUE THREAD

**Issue by carolyncole** [psu-libraries/scholarsphere-3#464]

**Comment by mtribone** [comment]
This is for the thumbnails created by the system and currently returns the file name in the alt text and is not useful for people who use assistive technologies. Recommend aria_hidden or alt=""

**Comment by mtribone** [comment]
http://csskarma.com/blog/difference-rolepresentation-aria-hiddentrue/
